.github/workflows/deploy.yml

```diff
@@ -42,19 +42,19 @@ jobs:
           #
           # Default: false
           submodules: recursive
-      
+
       - name: Build schema and guidelines with Docker image
         run: docker run --rm -v $(pwd):/opt/docker-mei/music-encoding ghcr.io/music-encoding/docker-mei:latest ant -noinput -buildfile music-encoding/build.xml -Ddocker=true -Dgithub.sha=${{ github.sha }} -Dgithub.event_name=${{ github.event_name }} -Dgithub.ref=${{ github.ref }}
 
-      ### UPLOADING THE ARTIFACTS ### 
+      ### UPLOADING THE ARTIFACTS ###
       - name: Upload PDF
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v3.1.2
```

Comment:
Why change this? `@v3` will take the latest version 3 anyway.

Reply:
```suggestion
        uses: actions/upload-artifact@v3
```